apps/condo/domains/acquiring/components/payments/PaymentsTable.tsx

```diff
@@ -200,6 +201,19 @@ const PaymentsTableContent: React.FC<IPaymentsTableProps> = ({ billingContext, c
         }
     }, [])
 
+    /**
+    * set default filters only when filters are empty and reset button hasn't been pressed
+    **/
+    useEffect(()=>{
+        if (hasFilters){
+            filtersAreReset = false
```

Comment:
using variables of outer scoupe can cause hard to find misbehaviour if you use several instances of this component - they will affect each other. please use this variable inside using setState